app/views/details/index.njk

```diff
@@ -25,8 +25,8 @@
       <div class="govuk-grid-column-two-thirds">
         <p> </p>
       </div>
-      <div class="govuk-grid-column-one-third">
-        <a id='downloaddetailsLink' class='mpdp-link-btn nojs-hidden'
+      <div class="govuk-grid-column-one-third mpdp-justify-content-flex-end">
+        <a id='downloaddetailsLink' class='mpdp-link-btn nojs-hidden mpdp-download-margin-botton'
```

Comment:
You can avoid adding a single css property class for margin as we have access to govuk design system specified responsive spacing. 
So for margin-bottom: 10px, you can use `govuk-!-margin-botton-2` css class. 

For more information check this page out:
https://design-system.service.gov.uk/styles/spacing/

Reply:
Agreed, made the changes as suggested.